src/components/layout/Navbar/NavGroup/NavGroup.tsx

```diff
@@ -19,6 +22,30 @@ const NavGroup = () => {
                 <NavBarLink path={value.path} linkText={value.text} key={key} />
               );
           })}
+
+          <NavBarLink
+            path={''}
+            linkText={'Create'}
+            key={'create-ant-nav-button'}
+            onClick={
+              walletAddress
+                ? () =>
+                    dispatchGlobalState({
+                      type: 'setShowCreateAnt',
+                      payload: true,
+                    })
+                : () => {
+                    dispatchGlobalState({
+                      type: 'setShowCreateAnt',
+                      payload: true,
+                    });
+                    dispatchGlobalState({
+                      type: 'setShowConnectWallet',
+                      payload: true,
+                    });
```

Comment:
what if within the `CreateAntModal` we check if there's a wallet address and if not, dispatch the state there

Reply:
Like in a use effect?

Reply:
yeah